source/Junkership.js

```diff
@@ -20,6 +20,7 @@ export default class Junkership extends Pixi.Sprite {
             this.y + 1 , // Top offset
             this.width - 3 , // Right offset + left offset
             this.height - 3 )// Bottom offset + top offset
+        this.WeaponList = [new PeaShoota(), new TriShoota(), new FiveShoota(), new RapidFire(), new SprayShot(), new SuperSprayShot(), new CrazySprayShot(), new VertSprayShot(), new VertShoota(), new RapidSprayShot(), new BFG()]
```

Comment:
@HolySchmitt and @dhowerton: So why aren't we just instantiating one powerup at a time? Like, when you get a powerup, instantiate that powerup, instead of instantiating aaaaaaall of them at once. You can just as easily have an array of classes as you can have an array of instances of those classes.